CONTRIBUTORS

```diff
@@ -37,3 +37,5 @@ Richard Coates
 THANOS SIOURDAKIS
 Toover
 Xavier Mehrenberger
+
+The manual drawings are from Coline, they are under CC BY 4.0.
```

Comment:
Hey, is it possible to have a full name here?

Reply:
I made it more precise indeed.